src/SolutionTemplate/UnoSolutionTemplate/iOS/Main.cs

```diff
@@ -1,15 +1,15 @@
 ﻿using UIKit;
 
-namespace $ext_safeprojectname$.iOS
+namespace $ext_safeprojectname$
 {
-	public class Application
+	public partial class App
```

Comment:
I'm not sure that merging the two classes is the best way to work there. Let's keep them separate.